manifests/kustomize/env/aws/accesskey/aws-configurations-patch.yaml

```diff
@@ -0,0 +1,93 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: workflow-controller-configmap
+  namespace: kubeflow
+data:
+  config: |
+    {
+    executorImage: $(executorImage),
+    artifactRepository:
+    {
+        archiveLogs: true,
+        s3: {
+            bucket: $(artifactRepositoryBucket),
+            keyPrefix: $(artifactRepositoryKeyPrefix),
+            endpoint: s3.amazonaws.com,
+            insecure: false,
+            accessKeySecret: {
+                name: $(awsSecretName),
+                key: $(awsAccessKeySecretKey)
+            },
+            secretKeySecret: {
+                name: $(awsSecretName),
+                key: $(awsSecretKeySecretKey)
+            }
+        }
+    }
+    }
+---
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: ml-pipeline-ui
+spec:
+  template:
+    spec:
+      containers:
+        - name: ml-pipeline-ui
+          env:
+            - name: AWS_ACCESS_KEY_ID
```

Comment:
@IronPan In future we should probably switch to getting the keys from ConfigMap.

Reply:
access_key_id and access_key sounds like they belong to a secret. I think this is appropriate.

Reply:
> access_key_id and access_key sounds like they belong to a secret.

AFAIK, they're not the secret itself, more like secret name.

P.S. If the Frontend currently gets those configurations from the ENV, then I guess this PR just reflects that. We should move them to configMap when we have a chance.